tests/main/system-usernames/task.yaml

```diff
@@ -8,7 +8,7 @@ environment:
     # List of expected snap install failures due to libseccomp/golang-seccomp
     # being too old. This should only reduce with time since new systems should
     # have newer libseccomp and golang-seccomp
-    EXFAIL: "centos-7-64 debian-9-64 fedora-29-64 fedora-30-64 opensuse-15\\.0-64 ubuntu-14"
+    EXFAIL: "centos-7-64 debian-9-64 debian-10-64 fedora-29-64 fedora-30-64 opensuse-15\\.0-64 ubuntu-14"
```

Comment:
Hmmm this is rather unexpected. I think debian 10 should carry libseccomp/golang-seccomp versions that are much more recent than ones in 9.